ci/cron/src/Main.hs

```diff
@@ -90,163 +89,90 @@ http_get url = do
-            else if old_version_exists
-            then do
-                -- Note: this checks for upload errors; this is NOT in any way
-                -- a protection against tampering at the s3 level as we get the
-                -- checksums from the s3 bucket.
-                putStrLn "  Found. Checking integrity..."
-                checksums_match <- checksums $ old </> version
-                if checksums_match
-                then do
-                    putStrLn "  Checks, reusing existing."
-                    copy (old </> version) $ new </> version
-                    return $ Just (gh_version, False)
-                else do
-                    putStrLn "  Check failed. Rebuilding..."
-                    build version new
-                    return $ Just (gh_version, True)
-            else do
-                putStrLn "  Not found. Building..."
-                build version new
-                return $ Just (gh_version, True))
-        putStrLn $ "Copying current (" <> current <> ") to top-level..."
-        copy (new </> current </> "*") (new <> "/")
-        putStrLn "Creating versions.json..."
-        let (releases, snapshots) = List.partition (not . prerelease . fst) documented_versions
-        create_versions_json (map fst releases) (new </> "versions.json")
-        create_versions_json (map fst snapshots) (new </> "snapshots.json")
-        -- Starting after 0.13.54, we have changed the way in which we trigger
-        -- releases. Rather than releasing the current commit by changing the
-        -- VERSION file, we now mark an existing commit as the source code for
-        -- a release by changing the LATEST file. This raises the question of
-        -- the release notes: as we tag a commit from the past, and keep the
-        -- changelog outside of the worktree (in commit messages), that means
-        -- that commit cannot contain its own release notes. We have decided to
-        -- resolve that conundrum by always including the release notes from
-        -- the most recent release in all releases.
-        case filter snd documented_versions of
-          ((newly_built,_):_) -> do
-              putStrLn $ "Copying release notes from " <> name newly_built <> " to all other versions..."
-              let p v = new </> name v </> "support" </> "release-notes.html"
-              let top_level_release_notes = new </> "support" </> "release-notes.html"
-              shell_ $ "cp " <> p newly_built <> " " <> top_level_release_notes
-              Data.Foldable.for_ documented_versions $ \(gh_version, _) -> do
-                  shell_ $ "cp " <> top_level_release_notes <> " " <> p gh_version
-          _ -> do
-              putStrLn "No version built, so no release page copied."
-        return new
+            build version
+            putStrLn $ "Pushing " <> version <> " to S3 (as subfolder)..."
+            push_to_s3 $ (temp </> version) version
```

Comment:
This looks like a typo, `temp </> version` is probably not a function .

Reply:
Looks like you _just missed_ the version that actually compiles by about three minutes. Sorry about that.